src/content/ja/2019/compression.md

```diff
@@ -0,0 +1,321 @@
+---
+part_number: IV
+chapter_number: 15
+title: Compression
```

Comment:
Should the title (Compression) be translated?

Seems a. bit of inconsistency in the Japanese chapter with some translated and some not:

```
accessibility.md:title: アクセシビリティ
caching.md:title: Caching
cdn.md:title: CDN
compression.md:title: Compression
css.md:title: CSS
http2.md:title: HTTP/2
markup.md:title: マークアップ
performance.md:title: パフォーマンス
pwa.md:title: PWA
seo.md:title: SEO
```

I think it probably makes sense not to translate specific technical terms (e.g. HTTP/2, PWA or SEO) but think generic terms (Cachign, Compression) probably should be translated. At least that's what the French team have done, for example:

```
caching.md:title: Mise en cache
cms.md:title: CMS
markup.md:title: Balisage Web
resource-hints.md:title: Indices de Ressources
third-parties.md:title: Tierces Parties
```

Reply:
Thanks, Misstaken.
I fix.

Reply:
Thanks. Should we also fix the Caching chapter? And are the others OK to leave in English as acronyms?

Reply:
Thanks, I fix caching title.

Sorry, I don't understand below.
"And are the others OK to leave in English as acronyms?"

Reply:
Technical terms and code should not be translated. Do the other titles fall into this category? For example CDN is short for Content Delivery Network so is it right to leave in English because it’s a technical term?

Looking at Wikipedia they leave it in English even in Japanese version: https://ja.wikipedia.org/wiki/Content_delivery_network so think we should do the same. So leave them.

So think we’re all done now.